### PR TITLE
emacsPackages.ebuild-mode: switch to melpaBuild

### PR DESCRIPTION
**Pull request by jian-lin**
Advantages:
- info manual is built and installed
- conform to package.el without any config, open a foo.ebuild file and its major-mode is set to ebuild-mode.

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix [hunk 1]
```diff
@@ -9,6 +9,14 @@ trivialBuild rec {
     sha256 = "sha256-vJ+UlPMIuZ02da9R67wIq2dVaWElu/sYmWx2KgBQ9B8=";
   };
 
+  commit = "not-used-but-has-to-be-set";
```
Comment (AndersonTorres) [inline pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix:12]:
```suggestion
  commit = "ef0991c2f0c385df5bbefbb2405f7873055556bc";
```

From their site:

https://gitweb.gentoo.org/proj/ebuild-mode.git/commit/?h=ebuild-mode-1.65

**Review by AndersonTorres — DISMISSED** [review]

File: pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix (hunk 1 above)
Comment (jian-lin) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix:12]: Why bother?  It builds fine. :)

Is [r-ryantm](https://github.com/r-ryantm) able to update `commit`?

Comment (AndersonTorres) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix:12]: It's possible with an updater script.

Comment (AndersonTorres) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix:12]: If this really builds fine, then `commit` is dead code and should be removed from all other packages using `melpaBuild`.

Comment (jian-lin) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix:12]: I will check the upstream MELPA build script and see if there is something we can do in a few days.

Comment (AndersonTorres) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix:12]: For now I will merge this, even not liking it.

**Pull request merged**